test/k8sT/assertionHelpers.go

```diff
@@ -53,3 +53,11 @@ func ExpectCEPUpdates(vm *helpers.Kubectl) {
 	err := vm.WaitCEPReady()
 	ExpectWithOffset(1, err).To(BeNil(), "CEP does not updated correctly")
 }
+
+// ExpectETCDOperatorReady is a wrapper around helpers/WaitForNPods. It asserts
+// the error returned by that function is nil.
+func ExpectETCDOperatorReady(vm *helpers.Kubectl) {
+	// We need to wait for all etcd pods to be ready (1 etcd-operator + 3 etcd nodes)
```

Comment:
You didn't change this comment, please add the following: 

```
Etcd operator creates 4 nodes (1 etcd-operator + 3 etcd nodes), the new pods are added when the previous is ready, so we need to wait until 4 nodes are in ready state. This is to avoid cases where a few pods are ready, but the new one is not  created yet. 
```